tensorflow_serving/batching/batching_session.cc

```diff
@@ -20,6 +20,8 @@ limitations under the License.
 #include "tensorflow/core/framework/tensor.h"
 #include "tensorflow/core/framework/tensor_shape.h"
 #include "tensorflow/core/framework/tensor_util.h"
+#include "tensorflow/core/framework/types.h"
```

Comment:
nit: I think you can remove these two #includes now that you've moved the type-specific handling to batching_util.